storage-node/packages/cli/src/commands/dev.ts

```diff
@@ -19,6 +19,62 @@ function developmentPort() {
   return 3001
 }
 
+const dispatchCalls = async (runtimeApi, senderAddress, rawCalls, batched = false) => {
+  const api = runtimeApi.api
+  const numCalls = rawCalls.length
+
+  debug(`processing ${numCalls} transactions...`)
+  let lastCall
+
+  for (let i = 0; i < numCalls; i++) {
+    debug(`${i + 1}/${numCalls} processed.`)
+    const { methodName, sectionName, args } = rawCalls[i]
+    const tx = api.tx[sectionName][methodName](...args)
```

Comment:
I purposely avoided adding any versioned store specific api methods in the RuntimeApi package because really the storage-node isn't aware of the versioned store.
As a matter of fact this whole dev-init script probably doesn't belong in the storage-node package, its become a much more high level script. Maybe we can factor it out later instead.